tests/epyccel/test_epyccel_modules.py

```diff
@@ -104,3 +104,20 @@ def test_module_5(language):
     max_pyt = mod.get_sum2()
     max_pyc = modnew.get_sum2()
     assert np.isclose( max_pyt, max_pyc, rtol=1e-14, atol=1e-14 )
+
+def test_module_6(language):
+    import modules.consts as mod
+
+    modnew = epyccel(mod, language=language)
+
+    atts = ('g', 'R0', 'rMin', 'rMax', 'skip_centre',
+            'method', 'compl', 'tiny')
+    for att in atts:
+        mod_att = getattr(mod, att)
+        modnew_att = getattr(modnew, att)
+        assert mod_att == modnew_att
+        #TODO: Remove .item() when #735 is fixed
+        assert type(mod_att) is type(modnew_att) or \
+                type(mod_att.item()) is type(modnew_att)
```

Comment:
You can remove `.item()` now, as #1052 was just merged.

Reply:
Done 2c27bcd